export_test.go

```diff
@@ -19,6 +19,48 @@
 
 package secboot
 
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+func (o *InitializeLUKS2ContainerOptions) CryptsetupArguments() []string {
```

Comment:
this matches corresponding code in luks2/internal, its complexity its a bit odd given is used in tests, the previous approach of spelling out things by hand seemed ok to me, so I'm probably missing something. Anyway if kept ExpectedCryptsetupArguments seems a bit of a clearer name for it

Reply:
Ah, I can't remember why I moved this, particularly because the next PR actually deletes this bit of code